### ISSUE TITLE
Collapsed guide navigation is not accessible

### ISSUE THREAD

**Issue by nickcolley** [alphagov/frontend#1116]
We need to update the pattern so it's similar to how we do our progressive disclosures, which adds attributes to tell assistive technologies what state it is in, expanded or not.

https://youtu.be/b5S6t_0ijY4

Seen at https://www.gov.uk/become-lorry-bus-driver

**Comment by tijmenb** [comment]
I think that the move of mainstream formats to government-frontend fixed this issue. Please reopen if that's not the case.
